quartz/styles/base.scss

```diff
@@ -400,6 +400,7 @@ code {
   border-radius: 5px;
   padding: 0.1rem 0.2rem;
   background: var(--lightgray);
+  overflow-x: scroll;
```

Comment:
this affects inline code as well, this should only apply for code inside pre

Reply:
I see, I have moved `overflow-x: scroll;` under code inside pre.